shaq_overflow/app/controllers/answers_controller.rb

```diff
@@ -0,0 +1,54 @@
+class AnswersController < ApplicationController
+
+  def index
+    @answers = Answer.all
+  end
+
+  def show
+    @answer = Answer.find(params[:id])
+  end
+
+  def new
+    @answer = Answer.new
+    @question = Question.find(params[:question_id])
```

Comment:
If the answer you just loaded belongs to this question, you could just do @question = @answer.question (or not bother as you'll have access to that in the view anyway.) The second query is probably not needed explicitly here.